src/runtime/execute_process_service.ts

```diff
@@ -165,34 +185,77 @@ export class ExecuteProcessService implements IExecuteProcessService {
     });
   }
 
+  private async _validateStartRequest(
+    requestingIdentity: IIdentity,
+    processModelId: string,
+    startEventId: string,
+    endEventId?: string,
+    waitForEndEvent: boolean = false,
+  ): Promise<void> {
+
+    const processModel: Model.Types.Process = await this._processModelUseCases.getProcessModelById(requestingIdentity, processModelId);
+
+    if (!processModel.isExecutable) {
```

Comment:
You could increase readability here like this:

```suggestion
    const processModelIsNotExecutable: boolean = processModel.isExecutable === false || processModel.isExecutable === undefined;
    if (processModelIsNotExecutable) {
```

Reply:
There really is no need for that. That flag is always a boolean and will always be either true or false.

In terms of readability I really don't see that much benefit in this.